app/assets/javascripts/components/overview/available_actions.jsx

```diff
@@ -13,21 +16,40 @@ const AvailableActions = React.createClass({
     current_user: React.PropTypes.object
   },
 
-  mixins: [CourseStore.mixin],
+  mixins: [CourseStore.mixin, ConfirmationStore.mixin],
 
   getInitialState() {
-    return getState();
+    return ({
+      course: CourseStore.getCourse(),
+      showConfirm: false,
```

Comment:
You can probably leave out all the null values in initial state. If not defined, they will all be falsey by default, so it should behave the same way without them.